db/src/logger.rs

```diff
@@ -239,8 +241,10 @@ impl Logger {
 
         if config.fs_locks {
             if config.fs_locks_block {
+                #[cfg(not(target_family = "wasm"))]
```

Comment:
I believe you can add this check to the `if` statement on 242 to get rid of the double cfg on both branches

Reply:
if someone tries to use `fs_locks` and is on `wasm` we should either panic or return an error (not silently continue to work)